guides/common/modules/con_smart-proxy-networking.adoc

```diff
@@ -0,0 +1,54 @@
+[id="{smart-proxy-context}-networking"]
+= {SmartProxy} networking
+
+The communication between {ProjectServer} and hosts registered to a {SmartProxyServer} is routed through that {SmartProxyServer}.
+{SmartProxyServer} also provides {Project} services to hosts.
+
+Integrated {SmartProxy}::
```

Comment:
This is a concept I'd love to get rid of. It's confusing people and serves no purpose IMHO.

We can wait until we containerize, but as it's looking now we will not provide a Foreman Proxy on the `{ProjectServer}`. At least, not by default.

Reply:
I think the section will still work without the definitions of integrated/external smart proxies. So that's what I did -- I dropped them.